alchemy/package.json

```diff
@@ -114,6 +115,7 @@
   },
   "dependencies": {
     "@aws-sdk/credential-providers": "^3.0.0",
+    "@clack/prompts": "^0.11.0",
```

Comment:
CLI dependencies should be in devDependencies and bundled in unless we are also using them in the core lib